README.md

```diff
@@ -4,6 +4,37 @@
 
 A log4j vulnerability filesystem scanner and Go package for analyzing JAR files.
 
+## Installation
+As of right now you have the following two options to install log4jscanner:
+
+### Go Install
+This requires Go to be installed on your system. You can find instruction how to
+install Go in the [Go documentation](https://go.dev/doc/install).
+
+Once Go is installed, you can use the following command to install log4jscanner:
+
+```
+go install github.com/google/log4jscanner@latest
+```
+
+### Build from source
+You can build the binary from source. This also requires Go to be installed as
+a precondition. You can find instruction how to install Go in the
+[Go documentation](https://go.dev/doc/install).
+
+Once Go is installed, you can use the follwing commands to build log4jscanner
```

Comment:
Small typo, should be "following" instead of "follwing"

Reply:
It should certainly be.
Thanks, will fix it.